extensions/dependencies/ui_leaflet.html

```diff
@@ -0,0 +1,4 @@
+<link rel="stylesheet" href="/third_party/static/leaflet-1.4.0/leaflet.css">
+<script src="/third_party/static/leaflet-1.4.0/leaflet.js"></script>
+<script src="/third_party/static/angular-ui-leaflet-1.0.3/ui-leaflet.min.no-header.js"></script>
+<script src="/third_party/static/angular-simple-logger-1.0.3/angular-simple-logger.min.js"></script>
```

Comment:
Why do we need "simple logger"? Can we do without it?

Reply:
This is needed as a dependency for ui-leaflet (I have added a comment above this line.)